tool/src/main/java/org/datacommons/tool/Main.java

```diff
@@ -58,6 +59,21 @@
       description = "Resolves local references and generates node DCIDs.")
   public boolean doResolution;
 
+  // TODO: Default to true after some trials.
+  @CommandLine.Option(
+      names = {"-s", "--stat-checks"},
+      defaultValue = "false",
+      scope = CommandLine.ScopeType.INHERIT,
+      description =
+          "Checks integrity of time series by checking for holes, variance in values, etc.")
+  public boolean doStatChecks;
+
+  @CommandLine.Option(
+      names = {"-p", "--sample-places"},
+      scope = CommandLine.ScopeType.INHERIT,
+      description = "List of place dcids to run stats check on.")
```

Comment:
I don't think the dependency needs to be asserted because if user enters sample places without enabling statchecks, it doesn't do anything, the sample places will just go unused. I added a log warning that alerts users that their sample places will go unused, do you think that's good enough? Also updated the description of the sample places option

Reply:
sg